src/components/mindsets/Mindsets.js

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment, useEffect } from "react"
+import PropTypes from "prop-types"
+import { connect } from "react-redux"
+
+// Components
+import Spinner from "../layout/Spinner"
+import {
+  MindsetList
+} from './'
+
+// Redux
+import { loadMindsets } from "../../actions/mindsetActions"
+
+// Material UI
+import Typography from "@material-ui/core/Typography"
+import Container from "@material-ui/core/Container"
+
+const Mindsets= ({ loadMindsets, mindset }) => {
+  const {
+    loading,
+    error,
+    all,
+  } = mindset
+
+  useEffect(() => {
+    loadMindsets()
+  }, [loadMindsets])
+
+  if (loading) {
+    return <Spinner />
+  } else if (error) {
+    return (
+      <Fragment>
+        <Container maxWidth="lg">
+          <Typography variant="h1">Your Compensation Mindset</Typography>
+          <Typography variant="body1">
+            There was a problem loading your results! Try again?
+          </Typography>
+        </Container>
+      </Fragment>
```

Comment:
I wanted to handle the error case here, but I don't really have a sensible error message. Might need someone who's a better wordsmith.